Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/Instance/InstanceUpdateExecutorInterface.h

```diff
@@ -34,6 +34,11 @@ namespace AzToolsFramework
             // A notification OnRootPrefabInstanceLoaded will fire during the propagation if root
             // prefab instance is loaded for the first time after this function is called.
             virtual void QueueRootPrefabLoadedNotificationForNextPropagation() = 0;
+
+            // Indicates whether to pause instance propagation or not.
+            //! When making property changes in the entity editor, pausing propagation during
+            //! editing will prevent the user from losing control of the properties they are editing.
+            virtual void SetShouldPauseInstancePropagation(bool shouldPausePropagation) = 0;
```

Comment:
nit: Part of the comment does not have doxygen notation.

Also, the function is a setter, not a getter, so "Indicates" is confusing. Maybe it could be `Sets whether to pause instance propagation or not.`?

Reply:
Oh yeah, will change!